generated/nidcpower/nidcpower.proto

```diff
@@ -93,6 +94,7 @@ service NiDCPower {
   rpc DisableSourceTriggerWithChannels(DisableSourceTriggerWithChannelsRequest) returns (DisableSourceTriggerWithChannelsResponse);
   rpc DisableStartTrigger(DisableStartTriggerRequest) returns (DisableStartTriggerResponse);
   rpc DisableStartTriggerWithChannels(DisableStartTriggerWithChannelsRequest) returns (DisableStartTriggerWithChannelsResponse);
+  rpc DiscoverCurrentRanges(DiscoverCurrentRangesRequest) returns (DiscoverCurrentRangesResponse);
```

Comment:
After you get past that failure, in the Draft PR I had to update the DC Power tests slightly as well because a param type changed (we think correctly): https://github.com/ni/grpc-device/pull/744/files#diff-02801be49181baef9324a74a76ee30cc3c41306f9b3c02b7465180b894c81df4

Reply:
Also, I'm making some updates to DC Power metadata to pull in new custom types so you can leave DC Power off this review and I can handle these with my changes.

Reply:
Marked as resolved. Removed `nidcpower` from this PR since Ryan will be updating it.

Reply:
FYI, I've included this change in [!372484](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/)